### PR TITLE
Create 2023-04-03 console release notes

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
"Console" release notes:
https://neon-next-git-dprice-04-02-console-relnotes-neondatabase.vercel.app/docs/release-notes/2023-04-03-console
"Plans" release note for Free Tier plan limit change:
https://neon-next-git-dprice-04-02-console-relnotes-neondatabase.vercel.app/docs/release-notes/2023-04-03-plans

- Minor edit to 03-15 "Plans" release notes to remove the "UI" tag, which doesn't really apply.
- Corrected the date for introduction of 100 hour limit
- Moved 100 hour limit release notes form Feb 6 to Feb 24 release note

### CODE REVIEW

File: content/release-notes/2023-04-02.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+label: 'Plans'
+---
+
+### What's new
+
+- Free Tier: The _compute active time_ limit of 100 hours per month for non-primary branches, introduced on February 6, 2023, now applies to all Neon Free Tier projects. The limit was not applied to projects created before February 6, 2023 to give users time to adjust their usage or upgrade to a [paid plan](https://neon.tech/pricing). For information about Free Tier limits, see [Free Tier](https://neon.tech/docs/introduction/technical-preview-free-tier). If you have questions about this change or upgrading to a paid plan, please contact [Sales](https://neon.tech/contact-sales).
```
Comment (nikitakalyanov) [inline content/release-notes/2023-04-02.md:7]: I think it's February 24, and not 6. Or am I missing something?

Comment (danieltprice) [reply content/release-notes/2023-04-02.md:7]: Hi @nikitakalyanov This looks like the release that introduced the 100 hours limit: https://github.com/neondatabase/cloud/pull/3795 
Did we actually implement the 100 hour limit on Feb 2nd? If you can confirm, I will update the dates.

Comment (danieltprice) [reply content/release-notes/2023-04-02.md:7]: Looks like the merge date for that release was Feb 6

Comment (nikitakalyanov) [reply content/release-notes/2023-04-02.md:7]: Yeah, but this merge date corresponds to the release to staging. The release to production happened on 24 of February. We also discussed it here: https://neondb.slack.com/archives/C04382GUYCC/p1678799568986299?thread_ts=1678792974.993889&cid=C04382GUYCC

Comment (danieltprice) [reply content/release-notes/2023-04-02.md]: Thank you. I will update here and a coupe of other places.

File: content/docs/introduction/technical-preview-free-tier.md [hunk 2]
```diff
@@ -20,6 +20,10 @@ The Neon Free Tier has the following limits:
 - 100 hours of _compute active time_ per month (total) on non-primary branches. When you reach the limit, compute endpoints assigned to [non-primary branches](/docs/reference/glossary#non-primary-branch) are unavailable until the limit resets at the beginning of the next month.
 - A point-in-time restore window of 7 days of _reasonable usage_.
 
+<Admonition type="note">
+On February 24, 2023, Neon removed the limit on the number of compute endpoints per project. The Free Tier now offers a compute endpoint for every branch and 100 hours of compute endpoint _active time_, as described above. The _active time_ limit is in effect for Neon projects created after this date. The limit will not be applied to Neon projects created before this date until April 1, 2023, to provide enough time for users to adjust their usage or upgrade to a paid plan. Neon launched paid plans in March, 2023. For information about our paid plans, refer to our [Pricing](https://neon.tech/pricing) page.
```
Comment (nikitakalyanov) [inline content/docs/introduction/technical-preview-free-tier.md:24]: It's good to have the correct date here, but this note almost duplicates the other note (see lines 10-12 in this file). Also, do we even need this note at all? I mean, it's already 5th of April, and old users are limited too

File: content/release-notes/2023-02-06-console.md [hunk 3]
```diff
@@ -12,7 +12,7 @@
...
 
-    Instead of limiting the number of endpoint computes in the Neon Free Tier, there is now a limit of 100 compute hours per month. Currently, the compute hour limit applies to newly created projects only. The limit does not yet apply to existing projects. We expect to apply the limit to all projects later this quarter. Regardless of the compute hour limit, you are always able to connect to the compute endpoint associated with the primary branch of your Neon project. For more information, see [Technical Preview Free Tier](/docs/introduction/technical-preview-free-tier).
+    Instead of limiting the number of endpoint computes in the Neon Free Tier, there is now a limit of 100 compute active time hours per month. Currently, the compute hour limit applies to newly created projects only. The limit does not yet apply to existing projects. We expect to apply the limit to all projects later this quarter. Regardless of the compute hour limit, you are always able to connect to the compute endpoint associated with the primary branch of your Neon project. For more information, see [Technical Preview Free Tier](/docs/introduction/technical-preview-free-tier).
```
Comment (nikitakalyanov) [inline content/release-notes/2023-02-06-console.md:15]: Hmm, I would expect this to be removed if we have the same info in the 24 Feb note (which is the correct date when this was rolled out to production)

File: content/docs/introduction/technical-preview-free-tier.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/introduction/technical-preview-free-tier.md]: Good point. The *Read more* link is gone from the Dashboard. Let's remove it entirely.

File: content/release-notes/2023-02-06-console.md (hunk 3 above)
Comment (danieltprice) [reply content/release-notes/2023-02-06-console.md]: Thank you. Meant to remove this one.

**Review by nikitakalyanov — APPROVED** [review]
Nice work, thanks!

**Pull request merged**